src/main/groovy/com/instabot/InstaBot.groovy

```diff
@@ -21,7 +21,10 @@ class InstaBot {
     def start(String loginUsername, String loginPassword) throws InterruptedException {
         try {
             LOG.info("Start InstaBot execution")
+            // TODO It's kind of weird that you instantiate the instaWebDriver here, then pass it as param and then just close it.
+            // I would extract the following and followers here, close the connection ASAP and pass them as parameters to RelatedUsersUpdater
             instaDriver = new InstaWebDriver(loginUsername, loginPassword)
+            //TODO lina_alexandrean should be a parameter from args, not hardcoded
```

Comment:
Agree.
I'm actually going to add a conf file that I can use for session-specific parameters.
e.g., loginUsername, password, masterUsername, and others. 
Meaning that I would store there the data that I would expect in the future to receive from the web-interface upon starting the process for a specific user. 

Please, let me know what do you think